src/vs/workbench/contrib/chat/browser/languageModelToolsService.ts

```diff
@@ -380,9 +380,11 @@ export class LanguageModelToolsService extends Disposable implements ILanguageMo
 		if (autoApproved) {
 			return;
 		}
-		const setting: { sound: 'auto' | 'on' | 'off' } = this._configurationService.getValue(AccessibilitySignal.chatUserActionRequired.settingsKey);
-		if (setting.sound === 'on' || (setting.sound === 'auto' && (this._accessibilityService.isScreenReaderOptimized() || !hasFocusedWindow))) {
-			this._accessibilitySignalService.playSignal(AccessibilitySignal.chatUserActionRequired, { customAlertMessage: this._instantiationService.invokeFunction(getToolConfirmationAlert, toolInvocations), userGesture: true });
+		const setting: { sound: 'auto' | 'on' | 'off'; announcement: 'auto' | 'off' } = this._configurationService.getValue(AccessibilitySignal.chatUserActionRequired.settingsKey);
```

Comment:
These should probably also have `| undefined` to be on the safe side?

Reply:
Mostly, technically it could be anything but you'd get a warning in the settings file and we generally aren't super safe and parse anything out.